sh/shadowenv.xonsh.in

```diff
@@ -0,0 +1,20 @@
+import json
+
+def __shadowenv() -> None:
+    shadow_data = $(@SELF@ hook --json)
```

Comment:
Instead of doing this as json, and processing that within the shell, this should be a `VariableOutputMode::XonshMode` in https://github.com/Shopify/shadowenv/blob/17f0bc93eadf021d197b5a25d4f02d1c7372b47e/src/hook.rs#L140

That way we keep as much as possible in precompiled space, making it quicker, which really matters with how often it will get called, and the context of it (the user just wants to get a prompt so they can run a command).

Reply:
Great suggestion! I'll get this adjusted 👍